libs/execution_context/ebpf_program.c

```diff
@@ -312,12 +312,15 @@ ebpf_program_create(ebpf_program_t** program)
         goto Done;
     }
 
+    retval = ebpf_object_initialize(
```

Comment:
As `ebpf_object_initialize` will insert the object in the global list, I think this should be called at the end of initialization.

Reply:
Done